packages/packages/commonwell-cert-runner/package.json

```diff
@@ -46,12 +46,12 @@
     "typescript": "^4.9.4"
   },
   "dependencies": {
-    "@metriport/commonwell-sdk": "^1.1.2",
+    "@metriport/commonwell-sdk": "^1.1.3",
     "commander": "^9.5.0",
     "dotenv": "^16.0.3",
     "lodash": "^4.17.21",
     "nanoid": "^3.3.4",
     "unique-names-generator": "^4.7.1"
   },
-  "gitHead": "98a769a26fff702be4f12814a78fc2344750b9e5"
+  "gitHead": "1c16cbb9f40a7cb58b4c0d4b7943f5c430d9cfb4"
```

Comment:
I know it was already there, but it seems we shouldn't be storing these on the repo:

> `gitHead` is a temporary property only meant to exist during lerna publish when it is constructing the package tarballs

https://github.com/lerna/lerna/issues/1880#issuecomment-456160357

https://stackoverflow.com/questions/57983774/lerna-publish-adds-githead-key-to-package-jsons

Apparently, this is a signal Lerna failed while doing its thing and left the package.json with a left-over... and we ended up pushing this to git.

Could just remove this from package.json and we try to avoid this in the future.